stack/2022082900/maximalocal.mac.template

```diff
@@ -0,0 +1,41 @@
+/* ***********************************************************************/
+/* This file is automatically generated at installation time.            */
+/* The purpose is to transfer configuration settings to Maxima.          */
+/* Hence, you should not edit this file.  Edit your configuration.       */
+/* This file is regularly overwritten, so your changes will be lost.     */
+/* ***********************************************************************/
+
+/* File generated on July 15, 2022, 4:29 pm */
```

Comment:
Have you ensured that the new maximalocal template (as shown on the healthcheck page) is the same as the old one (except for the templated variables)? Sometimes there can be some changes in the units.

Reply:
Yes. I took the template generated by the STACK healthcheck page and replaced all fixed paths with the variables. I just forgot to update the top comment block. I'll push an update ASAP.